docs/questions/sharing/visualizations/gauge.md

```diff
@@ -4,10 +4,57 @@ title: Gauge chart
 
 # Gauge chart
 
-Ah, **gauges**: you either love 'em or you hate 'em. …Or you feel "meh" about them, I guess. Whatever the case, gauges allow you to show a single number in the context of a set of colored ranges that you can specify. By default, when you choose the Gauge visualization, Metabase will create red, yellow, and green ranges for you.
+Ah, **gauges**: you either love 'em or you hate 'em. …Or you feel "meh" about them, I guess. Whatever the case, gauges allow you to show a single number in the context of a set of colored ranges that you can specify.
```

Comment:
Leaving this in? not sure about the tone

Reply:
Think that's a Maz original meta joke about gauges, we should leave it.

Reply:
oh my god i didn't even realize it was a meta joke...i love this now